tests/integration/component.go

```diff
@@ -340,6 +340,14 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "delete", "--all", "-f")...)
 
 		})
+
+		It("creates a local nodejs component without pushing and then deletes it using --all flag", func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+			helper.CmdShouldPass("odo", append(args, "create", "nodejs", "--project", project, "--context", context)...)
+			helper.CmdShouldPass("odo", append(args, "delete", "--all", "-f", "--context", context)...)
+			files := helper.ListFilesInDir(context)
+			Expect(files).NotTo(ContainElement(".odo"))
```

Comment:
You can add one more scenario like ```creates a local nodejs component without pushing and then deletes it using --all flag``` or you can also append these steps here
```
create component
push component
delete component using ```--all``` flag
Verify the component name should not be present in ```odo component list```
Verify the removal of component dir 
```

Reply:
I think pushing the component and deleting will add more time for the tests to complete. Also there is already a test to test the `--all` flag.

Reply:
> I think pushing the component and deleting will add more time for the tests to complete

@mik-dass After pushing the component and testing the --all flag is an important test scenario. We cannot scarifies the scenario in the name of time here atleast ;)

Reply:
> @mik-dass After pushing the component and testing the --all flag is an important test scenario. We cannot scarifies the scenario in the name of time here atleast ;)

Done